protocol/0085-RVST-rewards_vesting.md

```diff
@@ -0,0 +1,81 @@
+# Rewards Vesting Specification
```

Comment:
@cdummett can you point me to where the initial delay to vesting start is in the spec?